### ISSUE TITLE
CLI error messages are insufficient

### ISSUE THREAD

**Issue by vpillinger** [readmeio/swagger-inline#69]
Currently, the error messages when there is a formatting issue look like the following:

`An error occured:
Error: YAMLException: can not read a block mapping entry; a multiline key may not be an implicit key at line 48, column 9:
        type: integer:
            ^
    at /.../node_modules/swagger-inline/build/swagger-inline.js:108:19
    at Array.forEach (<anonymous>)
    at /.../node_modules/swagger-inline/build/swagger-inline.js:88:25
    at tryCatcher (/.../node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/.../node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/.../node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/.../node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/.../node_modules/bluebird/js/release/promise.js:693:18)
    at Promise._fulfill (/.../node_modules/bluebird/js/release/promise.js:638:18)
    at Promise._resolveCallback (/.../node_modules/bluebird/js/release/promise.js:432:57)
    at Promise._settlePromiseFromHandler (/.../node_modules/bluebird/js/release/promise.js:524:17)
    at Promise._settlePromise (/.../node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/.../node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/.../node_modules/bluebird/js/release/promise.js:693:18)
    at Promise._fulfill (/.../node_modules/bluebird/js/release/promise.js:638:18)
    at MappingPromiseArray.PromiseArray._resolve (/.../node_modules/bluebird/js/release/promise_array.js:126:19)
`
The issue is that it
1. Does not tell me which file had the issue
2. Does not point me to a place in the YAML that I need to fix.

I was forced to add a console.log statement into extractor.js to get a better error message

```
function buildEndpoint(route, yamlLines) {
  var endpoint = {};

  if (route) {
    console.log(yamlLines.join('\n')); // <-- Console.log here!
    var yamlObject = jsYaml.load(yamlLines.join('\n'));
    endpoint.method = route[1];
    endpoint.route = route[2];

    if (route[3]) {
      endpoint.summary = route[3];
    }

    Object.assign(endpoint, yamlObject);
  }

  return endpoint;
}
```

Now the CLI prints out the YAML for me to find the issue, but it also breaks the doc output.

**Comment by erunion** [comment]
If you're up for taking a stab at improving it, I'd be more than happy to pull it in!

**Comment by vpillinger** [comment]
That should improve the error messages.

Now it prints the route/schema in which a YAML error occurred in so an end-user can go right to the source of the issue.

**Comment by erunion** [comment]
Published to 3.0.3, thank you!
